runtime/src/disable.rs

```diff
@@ -0,0 +1,56 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{Drivers, MailboxResp};
+use caliptra_common::keyids::{KEY_ID_RT_CDI, KEY_ID_RT_PRIV_KEY, KEY_ID_TMP};
+use caliptra_drivers::{
+    hmac384_kdf, Array4x12, CaliptraError, CaliptraResult, Ecc384PrivKeyOut, KeyReadArgs, KeyUsage,
+    KeyWriteArgs,
+};
+
+pub struct DisableAttestationCmd;
+impl DisableAttestationCmd {
+    pub(crate) fn execute(drivers: &mut Drivers) -> CaliptraResult<MailboxResp> {
+        drivers
+            .key_vault
+            .erase_key(KEY_ID_RT_CDI)
+            .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
+        drivers
+            .key_vault
+            .erase_key(KEY_ID_RT_PRIV_KEY)
+            .map_err(|_| CaliptraError::RUNTIME_DISABLE_ATTESTATION_FAILED)?;
+        Self::generate_dice_key(drivers)?;
+        Ok(MailboxResp::default())
+    }
+
+    fn generate_dice_key(drivers: &mut Drivers) -> CaliptraResult<()> {
```

Comment:
Can you add a comment to this function explaining that this key is derived from an empty CDI slot, so it will not match the key that was certified in the Runtime Alias certificate?